docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -273,3 +273,9 @@ Make sure that the {agent}s assigned to this policy all have connectivity to the
 that you added. Lack of connectivity will prevent the {agent}
 from checking in with the {fleet-server} and receiving policy updates, but the agents
 will still forward data to the cluster.
+
+[discrete]
+[[agent-policy-scale]]
+== Scale of Policies Supported
+
+Maximum number of Elastic Agent policies supported by {fleet} is 500. You may experience slowness in response when a higher number of policies are configured. 
```

Comment:
```suggestion
The maximum number of {agent} policies supported by {fleet} is 500. If more than 500 policies are configured, {fleet} UI performance might be affected.
```
"You may experience slowness" is grammatically vague (the user does not experience slowness).